CMakeLists.txt

```diff
@@ -18,4 +18,4 @@ if (OpenCL_FOUND)
 	add_subdirectory(src/libethash-cl)
 endif()
 add_subdirectory(src/benchmark EXCLUDE_FROM_ALL)
-add_subdirectory(test/c EXCLUDE_FROM_ALL)
+add_subdirectory(test/c)
```

Comment:
So .. the only way to get the tests to find, compile and link to the ETHASH lib is to run cmake as specified in https://github.com/ethereum/ethash/blob/master/test/c/test.sh

If you do that in Windows the generated Visual studio project does not contain the Test project at all. Removing this allows for the Test project to show up in Visual Studio. 

Perhaps there is a better way with CMake? Or perhaps this whole thing is pretty wrong for anything other than Ubuntu?